.github/workflows/develop_main_mirror.yaml

```diff
@@ -0,0 +1,30 @@
+name: Mirroring
+
+# triggers a github action everytime there is a push or mr
+on:
+  push:
+    branches:
+      - develop
+      - main
+
+jobs:
+  # To test on HPC resources we must first mirror the repo and then trigger a pipeline
+  to_pnnl_gitlab:
+     # Latest Ubuntu as of Feb 2023 is 20.04
+     runs-on: ubuntu-22.04
+     steps:
+       # Action checks-out your repository under $GITHUB_WORKSPACE, so your workflow can access it.
+       - uses: actions/checkout@v1
+       #  Action for mirroring your commits to a different remote repository
+       - uses: yesolutions/mirror-action@master
+       # Give actions access to some secrets
+         with:
+            GIT_PUSH_ARGS: '--tags --force'
+            REMOTE: 'https://gitlab.pnnl.gov/exasgd/frameworks/exago-github-mirror'
```

Comment:
Does this repo exist? Should I be able to see it?

Reply:
Yeah it's there. Not sure who all will need access but it should be given to everyone a part of the ExaSGD group on GitLab I would hope